packages/core/hooks/useFileAccessContextMenu.ts

```diff
@@ -68,6 +78,26 @@ export default (filters?: FileFilter[], onDismiss?: () => void) => {
                         items: openWithSubMenuItems,
                     },
                 },
+                ...(isQueryingAicsFms && !isOnWeb
+                    ? [
+                          {
+                              key: "go-to-file-location",
+                              text: "Go to file location",
+                              title: "Show selected file in your native file browser",
+                              disabled:
+                                  fileDetails?.localPath === null ||
+                                  fileDetails?.localPath === undefined ||
+                                  fileSelection.count() > 1 ||
+                                  (!filters && fileSelection.count() === 0),
+                              iconProps: { iconName: "FolderOpen" },
```

Comment:
I only figured out I could use this icon by luck. I still have no idea where I can find valid Fluent UI icons.

Brian W. showed me this:
https://react.fluentui.dev/?path=/docs/icons-catalog--docs

But it would seem `FolderOpen` is not in their list.
It _does_ appear here but the icon on the list doesn't match what I get...
https://github.com/microsoft/fluentui-system-icons/blob/main/icons_regular.md

Reply:
https://developer.microsoft.com/en-us/fluentui#/styles/web/icons#fabric-react